### DISCUSSION TITLE
The links to files are pointing to my homestead file location and therefore not working.

### DISCUSSION THREAD

**Discussion by amacsmith** [spatie/laravel-ray#74] [Q&A] [answered]
I am using windows 10, homestead, and phpstorm. All the links to files that have been generated are like '/home/vagrant/project/path-to-file' where I need it to be an absolute path to my local host machine. How can I do that. Thanks by the way for the cool tool.

**Comment by freekmurze** [comment]
Hi, did you check out the Homestead page in our docs?

**Reply by amacsmith** [reply]
Yes

**Comment by amacsmith** [comment, marked as answer]
Yes, I did eventually get it to use my local_path though it seemed to have weird issues like C:/Users/amacsmith/dev\project\path-to-file.php. However, over time it did eventually seem to work. The other thing was that windows does not associate the phpstorm protocol to php storm and gives a blank "Please choose a default application" dialog box. Which I was able to work around by using my answer given in discussion #65. Which I hope will help any other windows 10 users facing this issue. Thanks for your time and appreciate the tool!
